src/utilities.js

```diff
@@ -107,13 +107,34 @@ export function getResults(searchType, obj) {
     },
   };
 
+  // Create array of items to query
+  var comparisonItems = [];
+
+  // If we are requesting an array of keywords for comparison
+  if (typeof obj.keyword === 'object' && obj.keyword.length) {
+    // Map the keywords to the items array
+    obj.keyword.map((keyword) => {
+
+      // Clone the original obj to keep the dates etc
+      let obj2 = JSON.parse(JSON.stringify(obj));
```

Comment:
I'm following [airbnb's javascript](https://github.com/airbnb/javascript#objects--rest-spread) guide.  Instead of doing `JSON.parse(JSON.stringify` to copy an object, use `{...obj}` (should be fine because  we only need a shallow copy)